### DISCUSSION TITLE
How to prevent files from being added to a static site?

### DISCUSSION THREAD

**Discussion by ghost** [vuejs/vitepress#3683] [Q&A] [answered]
For example, there are some json files in my project. How can I prevent them from being accessed? I have tried srcExclude with no success.

**Comment by brc-dd** [comment]
Are those files stored in public directory? If not then they won't be included in dist in production build.

**Reply by ghost** [reply]
They are in the same directory as markdown.

**Reply by brc-dd** [reply, marked as answer]
Try running vitepress build then preview command. Those JSON files won't be directly accessible. You don't need to configure anything.

**Reply by ghost** [reply]
You are right. I tried what you said and the JSON file cannot be accessed.Thank you so much!

It is recommended to note this situation in the document. The behaviour of the dev command and the build command is inconsistent~.
